etc/containerbuddy.json

```diff
@@ -1,5 +1,9 @@
 {
   "consul": "{{ .CONSUL }}:8500",
+  "onStart": [
```

Comment:
It's funny to use the old config names, but it's an old version of ContainerPilot (so old it's `containerbuddy`)